app/assets/sass/main.scss

```diff
@@ -5,6 +5,14 @@
 // Add your custom CSS/Sass styles below... 
 ///////////////////////////////////////////
 
+.nhsuk-icon__twitter{
+  fill: #00acee;
+}
+
+.nhsuk-icon__facebook{
+  fill: #3b5998;
```

Comment:
would the social icons be best styled to the style of each network (blue for Facebook and Twitter but green for Whatsapp etc.) or all styled NHS blue?